File: mRemoteV1/CS/Connection/Connection.Protocol.HTTPS.cs


namespace mRemoteNG.Connection.Protocol
{
	public class HTTPS : HTTPBase
	{
				
		public HTTPS(RenderingEngine RenderingEngine) : base(RenderingEngine)
		{
		}
				
		public override void NewExtended()
		{
			base.NewExtended();
			httpOrS = "https";
			defaultPort = (int)Defaults.Port;
		}
				
		public enum Defaults
		{
			Port = 443
		}
	}
}

File: mRemoteV1/Connection/Connection.Protocol.HTTPS.vb
Namespace Connection

    Namespace Protocol
        Public Class HTTPS
            Inherits HTTPBase

            Public Sub New(RenderingEngine As RenderingEngine)
                MyBase.New(RenderingEngine)
            End Sub

            Public Overrides Sub NewExtended()
                MyBase.NewExtended()

                httpOrS = "https"
                defaultPort = Defaults.Port
            End Sub

            Public Enum Defaults
                None = 0
                Port = 443
            End Enum
        End Class
    End Namespace

End Namespace
